internal/controllers/clusterextension_controller.go

```diff
@@ -47,6 +47,7 @@ import (
 	"github.com/operator-framework/api/pkg/operators/v1alpha1"
 	catalogd "github.com/operator-framework/catalogd/api/v1"
 	helmclient "github.com/operator-framework/helm-operator-plugins/pkg/client"
+	tokengetter "github.com/operator-framework/operator-controller/internal/authentication"
```

Comment:
```suggestion
	"github.com/operator-framework/operator-controller/internal/authentication"
```

I think we don't need the alias here right?
Can we keep the original import?

Reply:
fixed